src/Core/Customer/Application/Resolver/CreateTypeMutationResolver.php

```
@@ -0,0 +1,47 @@
+    }
+
+    /**
+     * @param array{
+     *     args: array{
+     *         input: array{
+     *             value: string
+     *         }
+     *     }
+     * } $context
+     */
+    public function __invoke(?object $item, array $context): CustomerType
```

PHPMD (2.15.0)

Avoid unused parameters such as '$item'. (undefined)

(UnusedFormalParameter)